Scripts/Proxies/ProxyAnimator.cs

```diff
@@ -27,26 +30,36 @@ public void Apply(Transform transform)
     [SerializeField]
     ActionMap m_ProxyActionMap;
 
-    ProxyHelper.ButtonObject[] m_Buttons;
+    [SerializeField]
+    ProxyAffordanceMap m_AffordanceMap;
```

Comment:
Should this just come from the proxy helper rather than being another serialized reference?  We have to maintain this link in here and the proxy UI right now

Reply:
Converting to m_AffordanceMapOverride.  Will default to using the override on ProxyAnimator & ProxyUI, if assigned, or use the default affordance map assigned to the proxy helper.